packages/x-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.2.1",
+  "version": "6.3.1",
```

Comment:
IMHO, it's a bit pointless releasing packages just because some dependency has been bumped, but no real user value has been added... 🙈 🤷 
The same does apply to the `x-license-pro` package.
But on the other hand, doing so ensures that anyone using these packages gets a cleaner resulting `lock` file, because of little to no out-of-sync sub-dependencies between different packages. 🤔 
Last week it was definitely pointless releasing the package, due to only the `tsconfig` being changed, but this week there has been a `yargs` package bump...

The bigger question is the `x-license-pro` package, which is clearly usually released only when there are actual changes in the underlying package functionality...
Maybe following the same approach for all packages makes sense? 🤔 
I believe `babel` is doing something similar—they do not release a package just to avoid different sub-package versions being used by various packages.

TL.DR.: I wouldn't mind only releasing our secondary packages (license, codemod), when they do include functional changes or other packages have moved to a different sub-package major. (i.e. `x-data-grid` has bumped `@babel/runtime` to v8.x, then releasing `x-license-pro` only with a bump is a must to avoid end-users having to install two majors of `@bael-runtime`).

Reply:
I agree, we can skip codemod and license packages in this release 👍

Reply:
I reverted the version of codemod and license packages